tests/metrics/test_stratification.py

```diff
@@ -1,371 +1,252 @@
+import itertools
 from datetime import datetime
 from math import floor
 from typing import Callable, List, Set
 from unittest.mock import MagicMock, patch
 
+import numpy as np
 import pandas as pd
 import pytest
+from vivarium import InteractiveContext
+from vivarium.framework.engine import Builder
+from vivarium.framework.population import SimulantData
+from vivarium.testing_utilities import TestPopulation, metadata
 
 from vivarium_public_health.metrics.stratification import (
-    ResultsStratifier,
+    ResultsStratifier as ResultsStratifier_,
+)
+from vivarium_public_health.metrics.stratification import (
     Source,
     SourceType,
     StratificationLevel,
 )
 
-AGE_BINS = pd.DataFrame(
```

Comment:
The purpose of all those test utilities is to make it so you don't need to write a ton of mock code to test new components like what's here.